### DISCUSSION TITLE
hi there Any tip on how to debug this [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2865] [Discord Threads]
> 🙋‍♂️ **ybon**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### Hey, looks like we miss access control for the endpoint. Thank you for reporting this. I've raised an issue for you you can follow updates from there (https://github.com/nocodb/nocodb/issues/2740).  
#  
> 🙋‍♂️ **ybon**  
#### Thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/998972287539740762)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
